packages/bpk-docs/src/pages/BannerAlertsPage/BannerAlertsPage.js

```diff
@@ -84,6 +84,81 @@ BpkBannerDismissable.defaultProps = {
   type: null,
 };
 
+// eslint-disable-next-line react/no-multi-comp
+class BpkBannerAlertDismissDemo extends Component {
+  constructor() {
+    super();
+
+    this.newBannerAlerts = [
+      {
+        show: true,
+        message: 'Neutral alert with dismiss option.',
+        type: ALERT_TYPES.NEUTRAL,
+      },
+      {
+        show: true,
+        message: 'Successful alert with dismiss option.',
+        type: ALERT_TYPES.SUCCESS,
+      },
+      {
+        show: true,
+        message: 'Warn alert with dismiss option.',
+        type: ALERT_TYPES.WARN,
+      },
+      {
+        show: true,
+        message: 'Error alert with dismiss option.',
+        type: ALERT_TYPES.ERROR,
+      },
+    ];
+
+    this.reset = this.reset.bind(this);
+    this.setDismissed = this.setDismissed.bind(this);
+
+    this.state = {
+      bannerAlerts: JSON.parse(JSON.stringify(this.newBannerAlerts)),
+    };
+  }
+
+  setDismissed(index) {
+    const updatedBannerAlerts = JSON.parse(JSON.stringify(this.state.bannerAlerts));
+    updatedBannerAlerts[index].show = false;
+    this.setState({
+      bannerAlerts: updatedBannerAlerts,
+    });
+  }
+
+  reset() {
+    this.setState({
+      bannerAlerts: JSON.parse(JSON.stringify(this.newBannerAlerts)),
+    });
+  }
+
+  render() {
+    return (
+      <div>
+        <BpkButton
+          onClick={this.reset}
+        >
+          Reset
+        </BpkButton>
```

Comment:
I feel that we should make this only appear when they've removed one or more of the alerts, as otherwise it won't have a function. We can just have a state variable called `objectsEdited` which starts `false`, becomes true when `setDismissed` is called and false again when `reset` is called. 

What do you think?